### ISSUE TITLE
Fix rspec tests

### ISSUE THREAD

**Issue by lauramosher** [Terrastories/terrastories#461]
Right now, if you run `docker-compose run web-test rspec`, there are a number of failing specs.

```
Finished in 14.18 seconds (files took 3.24 seconds to load)
84 examples, 21 failures
```

This issue is to work through those failing specs and make them pass.

**Comment by slaloggia** [comment]
I'll take this! A lot of them appear to be due to a fix I implemented for a different issue anyway, so I should have updated the specs then.

**Comment by slaloggia** [comment]
The .import_csv tests were failing bc the Place model is validating lat & and long. Should those be allowed to be blank or do we only want Places to be created with a latitude and longitude?

**Comment by lauramosher** [comment]
> Should those be allowed to be blank or do we only want Places to be created with a latitude and longitude?

I would say allowed to be blank but I am not sure what the original ticket was asking. @Terrastories/terrastories-stewards pinging to see if you have any input on the preference for lat/long requirements.

**Comment by slaloggia** [comment]
Is there anything else that wasn't addressed by that PR, or can this be closed?

**Comment by lauramosher** [comment]
@slaloggia This can be closed. I'm so used to them being closed automatically. 🙈 

In the future, you can tag a PR with one of the github keywords so it will automatically close this out. In the one linked above, instead of using "Address issue #461" you can use "fixed #461" or "closes #461." Here are all of the documented keywords that will close the associated issue: https://docs.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords

Thank you so much for your work on this!
